### ISSUE TITLE
Prompt for password on locked PDFs

### ISSUE THREAD

**Issue by 50Wliu** [izuzak/atom-pdf-view#92]
I'm working on password-protected PDFs currently and pdf-view doesn't seem to recognize that the PDF is locked.  Instead of prompting me for a password it just opens up a blank tab.

Warning in console:
`The provided value 'moz-chunked-arraybuffer' is not a valid enum value of type XMLHttpRequestResponseType.` at `pdf.worker.js:1650`

**Comment by izuzak** [comment]
Cool idea -- thanks for opening an issue, @50Wliu. I probably won't have time to look into this anytime soon, but happy to review a pull request if someone else wants to tackle it.

**Comment by 50Wliu** [comment]
pdf.js actually supports this...not sure why it isn't Just Working.
https://github.com/mozilla/pdf.js/blob/7a8b0fb107706c2ecf5539cfdfad63b3968700d0/web/password_prompt.js

**Comment by izuzak** [comment]
Yeah, no idea either. If someone wants to take a look and submit a pull request -- that would be :sparkles: Thanks for sharing that link! :bow:

**Comment by amattu2** [comment]
Any update on this?
